README.md

```diff
@@ -1,6 +1,6 @@
 # xhr
 
-A small xhr wrapper. Designed for use with [browserify](http://browserify.org/).
+A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/).
```

Comment:
Go all out and also link to MDN? - https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest